package.json

```diff
@@ -28,12 +28,12 @@
     "test:watch": "jest --watch",
     "lint": "eslint \"{src,test,scripts}/**/*.{js,jsx,ts,tsx}\" --quiet",
     "lint:fix": "npm run lint -- --fix",
-    "extract": "formatjs extract 'src/**/*.{js,jsx,ts,tsx}' --out-file temp.json --flatten --id-interpolation-pattern '[sha512:contenthash:base64:6]' --preserve-whitespace",
-    "compile": "formatjs compile 'temp.json' --out-file src/i18n/locales/en-US.json",
+    "extract": "formatjs extract \"src/**/*.{js,jsx,ts,tsx}\" --out-file temp.json --flatten --id-interpolation-pattern '[sha512:contenthash:base64:6]' --preserve-whitespace",
```

Comment:
The old version is working on mac - is it not working on windows with single quotes?

Reply:
Yep, it doesn't work on Windows.... Never worked I think.

Just now I got the "head" (insomnia... -.-) to deal with it.

Does this work on macOS this way?

Reply:
yes - this works on mac!